src/session.cpp

```diff
@@ -343,10 +350,8 @@ session_t::remote_endpoint() const {
         try {
             endpoint = ptr->socket->remote_endpoint();
         } catch(const std::system_error& e) {
-            // TODO: Log this.
+            // Ignore.
```

Comment:
I'm not sure, but I guess in this case it's okay since there's nothing you can really do or fix or whatever about it.